jest.config.js

```diff
@@ -0,0 +1,179 @@
+// For a detailed explanation regarding each configuration property, visit:
+// https://jestjs.io/docs/en/configuration.html
+
+module.exports = {
```

Comment:
This config is a first stab at getting a sensible starting point. I imagine this may get tweaked quite a bit as we find what actually works for us.